panels/models/attraction.py

```diff
@@ -663,24 +663,40 @@ def start_time_label(self):
         return 'unknown start time'
 
     @property
-    def checkin_time(self):
+    def checkin_start_time(self):
         advance_checkin = self.attraction.advance_checkin
         if advance_checkin < 0:
-            return self.end_time
+            return self.start_time
         else:
             return self.start_time - timedelta(seconds=advance_checkin)
 
     @property
-    def checkin_time_label(self):
-        checkin = self.checkin_time_local
+    def checkin_end_time(self):
+        advance_checkin = self.attraction.advance_checkin
+        if advance_checkin < 0:
+            return self.end_time
+        else:
+            return self.start_time
+
+    @property
+    def checkin_start_time_label(self):
+        checkin = self.checkin_start_time_local
+        today = datetime.now(c.EVENT_TIMEZONE)
```

Comment:
I wouldn't hold up this PR based on this minor stylistic nitpick, but ``today`` intuitively sounds like a ``datetime.date`` object, whereas ``now`` sounds more like a ``datetime.datetime`` object.